docs/index.md

```diff
@@ -2,9 +2,9 @@
 
 > Rendertron is a headless Chrome rendering solution designed to render & serialise web pages on the fly.
 
-- :hammer: Built with [Puppeteer](https://github.com/GoogleChrome/puppeteer)
-- :cloud: Easy deployment to Google Cloud
-- :mag: Improves SEO
+- ![](https://github.githubassets.com/images/icons/emoji/unicode/1f528.png) Built with [Puppeteer](https://github.com/GoogleChrome/puppeteer)
+- ![]https://github.githubassets.com/images/icons/emoji/unicode/2601.png?v8 Easy deployment to Google Cloud
+- ![]https://github.githubassets.com/images/icons/emoji/unicode/1f50d.png?v8 Improves SEO
```

Comment:
The second and third emoji are missing the parentheses around the URL. The first one has them, though :)

Reply:
Fixed the missing parentheses error. ✌️